Src/CSharpier.VSCode/src/CSharpierProcessPipeMultipleFiles.ts

```diff
@@ -20,9 +20,10 @@ export class CSharpierProcessPipeMultipleFiles implements ICSharpierProcess {
     }
 
     private spawnProcess = (csharpierPath: string, workingDirectory: string) => {
-        const csharpierProcess = spawn("dotnet", [csharpierPath, "--pipe-multiple-files"], {
+        const csharpierProcess = spawn(csharpierPath, ["--pipe-multiple-files"], {
```

Comment:
Uh... you sure about this?
What does `csharpierPath` evaluate to? a `.exe`? a `.dll`? I don't think dotnet produces standalone executables for any platform other than windows.

Reply:
It evaluates to "[path]/dotnet-csharpier", which works on windows because including the .exe is optional. I tested it out on ubuntu with when I did the rider plugin and just doublechecked it now for vscode in ubunto.